src/main/scala/org/tygus/suslik/synthesis/SynthesisRunnerUtil.scala

```diff
@@ -66,7 +66,7 @@ trait SynthesisRunnerUtil {
     val parser = new SSLParser
     val res = parser.parseGoal(text)
     if (!res.successful) {
-      throw SynthesisException(s"Failed to parse the input.")
+      throw SynthesisException(s"Failed to parse the input:\n$res")
```

Comment:
Does it really align well the syntax pointer given the added message prefix? Need a test to observe.

Reply:
Yes, because there are several new lines after the prefix. Message looks like this, for example:
```
Synthesis failed:
Failed to parse the input:
[27.87] failure: '`)'' expected but `{' found

{true ; lseg(x1, s1) ** lseg(x2, s2) ** ret :-> x2} void lseg_append (loc x1, loc ret {s =i s1 ++ s2 ; lseg(y, s) ** ret :-> y }
                                                                                      ^
```